README.md

```diff
@@ -66,6 +66,13 @@ Set up the environment:
     $ source venv/bin/activate
     $ pip install -r requirements/dev.txt
 
+Set environment variables for email interface settings:
```

Comment:
We need an email account to send an email from when we go live

Reply:
Just fyi - the production sites are already set up with email from accounts.

Reply:
@jmcarson if we set the `from_email` to `None` in the `django.core.mail.send_mail` method, it should send from the correct email, right? That's what it looks like the documentation is saying: https://docs.djangoproject.com/en/4.1/topics/email/#send-mail

Reply:
Yes, that is correct. If you wanted app users to have the flexibility to be able to use a different email for acm emails than the default site from address: DEFAULT_FROM_EMAIL then I would use another setting entirely but have it default to none, which then falls back to the default.

Reply:
Sounds good! Let's leave it as the default until we need something different.